src/main/java/org/opentripplanner/middleware/models/MobilityProfileLite.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.middleware.models;
+
+import java.util.Objects;
+
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+
+public class MobilityProfileLite {
+    public String userId;
+    public String mobilityMode;
+    public String email;
+    public String name;
```

Comment:
We will have to figure out how to handle the name, or if providing a nickname will be needed.